editions/tw5.com/tiddlers/widgets/ButtonWidget.tid

```diff
@@ -9,11 +9,11 @@ type: text/vnd.tiddlywiki
 
 The button widget displays an HTML `<button>` element that can perform a combination of optional actions when clicked:
 
-* Executing any ActionWidgets passed in the `actions` attribute
-* Executing any ActionWidgets that are immediate children of the button widget
+* Executing any ActionWidgets passed in the `actions` attribute (''best practice'')
+* Executing any ActionWidgets that are immediate children of the button widget (outdated -- use best practice instead)
```

Comment:
This kind of abbreviated language is unhelpful for users who don't have a good understanding of English. It would be clearer to say `Executing any ActionWidgets that are immediate children of the button widget, which is now deprecated in favour of using the actions attribute`.